package.json

```diff
@@ -141,6 +141,7 @@
     "@poi/plugin-eslint": "^12.0.0",
     "@types/poi": "^12.5.0",
     "babel-eslint": "^10.0.3",
+    "bunyan-pretty-cli": "github:vladimyr/bunyan-pretty-cli",
```

Comment:
So are we sticking with bunyan or are we we considering moving to something that is more maintained like `pino`?

Reply:
Well I made it to make sure we get pretty logs even something prevents us to go with pino. Using this is no brainer, switch to pino needs to be discussed more broadly but atm I'm all for it.

Reply:
👍